doc/decisions/README.md

```diff
@@ -46,3 +46,4 @@ section here.
 - [Global Validation](global_validation.md)
 - [Default Values](default_values.md)
 - [High Level API](high_level_api.md)
+- [Global KeySet](global_keyset.md)
```

Comment:
Why is it a draft? It is implemented?

Reply:
I'll update it when we add the cache. This will add info about cached vs volatile parts of the global keyset anyway. Thank you!